src/main/java/org/reactome/release/dataexport/configuration/ConfigurationManager.java

```diff
@@ -67,7 +67,7 @@ public ConfigurationManager(String configFilePath) {
 	 * configuration file if one needs to be recreated (this includes an exception from deleting an existing file,
 	 * writing a new file, or setting its file system permissions appropriately).
 	 */
-	public boolean createConfigurationFile(boolean overwriteExistingFile) throws IOException {
+	public boolean validateAndPotentiallyCreateConfigurationFile(boolean overwriteExistingFile) throws IOException {
```

Comment:
That's a mouthful! `validateOrCreate...` would be a little easier to read, but when a method name indicates that it can do two possible things, that might be an indication that the method can be refactored and split OR that the name is not really the right name.

Not a deal-breaker, this PR can be approved without further changes, but something to consider for the future.

Reply:
Yeah, I had a hard time thinking of a good name.  `validateOrCreate` is good.  I can refactor later on.